frontend/src/pages/ui/FeedbackBox.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useRef, useState } from "react";
+// @ts-ignore
+import Feedback from "@benhammondmusic/feeder-react-feedback";
+import "@benhammondmusic/feeder-react-feedback/dist/feeder-react-feedback.css"; // import stylesheet
+import { useBottomScrollListener } from "react-bottom-scroll-listener";
+import { useCookies } from "react-cookie";
+import useClickAway from "../../utils/useClickAway";
+import sass from "../../styles/variables.module.scss";
+
+const TEST_FEEDER_ID = "6171cc2965b82c00045239dc"; // this is a test account; we can create a new ID for our real dashboard account once secrets are working
+const FEEDBACK_ID = process.env.REACT_APP_FEEDBACK_ID || TEST_FEEDER_ID; // view collected feedback at feeder.sh/dashboard
```

Comment:
Replace this once we have our production and deploy test secrets working

Reply:
Nice